audit_exceptions/provided_by_macos_depends_on_allowlist.json

```diff
@@ -1,6 +1,7 @@
 [
   "apr",
   "apr-util",
+  "gnu-getopt",
```

Comment:
```suggestion
```

This shouldn't be added here. We can allowlist the CI failure instead.

Reply:
This is rebased on top of `breakerlib` because it is a common runtime dependency. Actual implementation is 38a9159660989ebbc14fe9c353e7fb4bce4ba945